### PR TITLE
add support for java opts when executing offline CLI

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

File: plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java [hunk 1]
```diff
@@ -177,7 +180,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         if (offline) {
             getLog().debug("Executing offline CLI scripts");
             try (StandardOutputStream out = StandardOutputStream.parse(stdout, false)) {
-                final int exitCode = offlineCLIExecutor.execute(jbossHome, getCommands(), getLog(), out, systemProperties);
+
+                String[] opts = javaOpts.split("[\\n\\s]+");
```
Comment (ctomc) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: This here is so we can support properties passes in multiple lines as well as just separated by space same as surefire does it

Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: `\s` already matches `\n`.

Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: From the JavaDoc:
> \s	A whitespace character: [ \t\n\x0B\f\r]

Comment (ctomc) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: Hmm, not sure why it gave me issues when I was testing it...

Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: Do you have an example string by chance? I'm just curious is all.

Comment (ctomc) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: https://github.com/wildfly/wildfly/blob/master/pom.xml#L6291

Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/cli/ExecuteCommandsMojo.java]: Thanks. With a simple test I see the same results with or without the `\n`. I'll give the actually build a test and see what happens though. Other than that I think this PR is fine and I'll get it merged here soon.

**Pull request merged**